libs/moment-adapter/README.md

```diff
@@ -0,0 +1,135 @@
+# @Fundamental-NGX/Core
```

Comment:
These are just the docs for fundamental-ngx/core, not the adapter

Reply:
we(me and @dimamarksman) decide to remove readme for adapter at all.